BaseClasses.py

```diff
@@ -336,6 +336,7 @@ def find_common_pool(players: Set[int], shared_pool: Set[str]) -> Tuple[
             region = Region("Menu", group_id, self, "ItemLink")
             self.regions.append(region)
             locations = region.locations
+            self.itempool.sort(key=lambda item: item.classification)
```

Comment:
I'm fine with this solution but I'd like a comment explaining it if that's ok :3

Reply:
Also, hmm, this does kind of ignore the possibility of people adding custom values to classification (which is intended & supported). This should probably just sort by `item.advancement`?